config/application.rb

```diff
@@ -56,8 +60,10 @@ class Application < Rails::Application
       end
     end
 
-    config.before_initialize do
-      Credentials.load_config
+    if Rails.env.development? || Rails.env.test?
```

Comment:
If we can avoid addition this conditional, that would be better. Ideally, if we want people to create their own development credentials, we can do that by providing a sample file, that with a rake task will generate the development key and the development encrypted credentials. Then all the custom environment wrangling can be removed.